src/lib/engine.js

```diff
@@ -103,6 +104,7 @@ assign(Engine.prototype, EventEmitter.prototype, {
       ship: this._ship,
       form: this._form,
       formIsSubmitted: this.formIsSubmitted(),
+      formIsExistent: this.hasForm(),
```

Comment:
Can we try to not keep the same name from the engine to the component please. If you do not like `hasForm`, what do you think of `formExists`?